plugin/src/test/resources/io/jenkins/plugins/redis/casc.yml

```diff
@@ -21,10 +21,13 @@
 # THE SOFTWARE.
 
 unclassified:
-  redis:
-    host: "test"
-    port: 3333
-    ssl: true
-    database: 3
-    connectionTimeout: 3
-    socketTimeout: 3
+  fingerprints:
+    fingerprintStorage:
```

Comment:
would it be better if this was just `storage`?

Reply:
I agree. So according to https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/PLUGINS.md#step-6, the way to do this should be this: https://github.com/jenkinsci/jenkins/pull/4834/files#diff-1d8d7d278d26766670b170520eb530fdR44 right? But it is not having any effect, I am still getting `fingerprintStorage` when I go to the view config page of CasC page.

Reply:
https://github.com/jenkinsci/jenkins/pull/4834#discussion_r453454117

Reply:
is this not using the latest incremental?

Reply:
Yeah, it is getting built currently on core